src/components/HomeCards/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import HomeCard from '../HomeCard/index';
+
+
+export default function HomeCards({panico}) {
```

Comment:
Try to define better names for your components, probably `videoCard` would be a better name. Also for your arguments if you don't have any idea about naming you can just name it `props`: [Here you can read about naming conventions](https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c#:~:text=Classes%20and%20objects%20should%20have,should%20not%20be%20a%20verb.&text=Method%20Names-,Methods%20should%20have%20verb%20or%20verb%20phrase,postPayment%2C%20deletePage%2C%20or%20save.)